node/tests/data_request_examples.rs

```diff
@@ -6,6 +6,7 @@ use std::{
 };
 use witnet_data_structures::chain::DataRequestOutput;
 use witnet_node::actors::messages::BuildDrt;
+use witnet_rad::script::RadonScriptExecutionSettings;
```

Comment:
fmt

Reply:
Why?

Reply:
I would combine the witnet_rad::script... and witnet_rad::types in witnet_rad::{}

Reply:
hahaha I was not seeing the next line from the issue comments view, sorry!